src/main/java/cholog/wiseshop/api/order/dto/OrderResponse.java

```diff
@@ -0,0 +1,21 @@
+package cholog.wiseshop.api.order.dto;
+
+import cholog.wiseshop.db.order.Order;
+import java.time.LocalDateTime;
+
+public record OrderResponse(Long id,
+                            Long productId,
+                            int count,
+                            LocalDateTime createdDate,
+                            LocalDateTime modifiedDate) {
```

Comment:
해당 response가 주문 단건 조회에 관한 response 맞을까요?
명세서에 productName 정보도 전달하기로 되어있었는데 없는 것 같아요! 혹시 제외한 별다른 이유가 있을까요?

```suggestion
public record OrderResponse(Long orderId,
                            Long productId,
                            int orderCount,
                            String productName,
                            LocalDateTime createdDate,
                            LocalDateTime modifiedDate) {
```
그리고, 변수명을 다음과 같이 명확하게 하는 건 어떨까요? (id->orderId, count -> orderCount // orderCount는 굳이 싶긴한데...그래도 일단 말해보아요)
id가 두 개 있어서 명확하게 하는 것도 좋을 것 같다는 생각도 들었고, 명세서랑 똑같이 가져가면 좋을 것 같아서요!

Reply:
> 해당 response가 주문 단건 조회에 관한 response 맞을까요?
명세서에 productName 정보도 전달하기로 되어있었는데 없는 것 같아요! 혹시 제외한 별다른 이유가 있을까요?

빠르게 CRUD만 만들어보자고 생각했어서 명세를 보지 않고 구현을 우선으로 진행했어요! 정책에 맞게 수정하면서 해당 부분도 반영해볼게요👍 

> 그리고, 변수명을 다음과 같이 명확하게 하는 건 어떨까요? (id->orderId, count -> orderCount // orderCount는 굳이 싶긴한데...그래도 일단 말해보아요)
id가 두 개 있어서 명확하게 하는 것도 좋을 것 같다는 생각도 들었고, 명세서랑 똑같이 가져가면 좋을 것 같아서요!

좋은 생각이네요👍
기존 `Entity`들의 id 값도 사실 `엔티티명+Id`로 진행하면 좋을거 같다고 생각했었어요! 해당 부분도 같이 대화해보고 기존 엔티티의 PK 값인 id도 저렇게 바꿀 수 있으면 좋겠네요!